### ISSUE TITLE
oXygen with Saxon EE 12.3 Results In Null Processor Exception

### ISSUE THREAD

**Issue by ajbufort** [expath/expath-http-client-java#50]
* oXygen version: <oXygen/> XML Editor 26.1, build 2024042206
* Saxon version: 12.3 EE
* EXPath HTTP Client version: 1.5.0
---

When running sample code from within oXygen to exercise the library (`http-client-saxon-1.5.0-uber.jar`) as directed in GitHub, I am getting the following error:

```
-- Cannot invoke "net.sf.saxon.s9api.Processor.newDocumentBuilder()" because "processor" is null. --
```

Oxygen run-time log captured and attached, as well as the XSL used in the test.

[oxygen_run_log.txt](https://github.com/user-attachments/files/15889567/oxygen_run_log.txt)
[sample_xsl.txt](https://github.com/user-attachments/files/15889614/sample_xsl.txt)

(XSL attached as txt since GitHub not supporting attachment of XSL files.)

Saxon config and uber-jar inclusion on classpath are correct as per GitHub directions for the client.

**Comment by adamretter** [comment]
Thanks @ajbufort I think the StackTrace might be:

```
java.lang.NullPointerException
	at org.expath.tools.saxon.model.SaxonTreeBuilder.<init>(SaxonTreeBuilder.java:38)
	at org.expath.httpclient.saxon.SaxonTreeBuilder.<init>(SaxonTreeBuilder.java:33)
	at org.expath.httpclient.saxon.SaxonResult.add(SaxonResult.java:117)
	at org.expath.httpclient.impl.HttpRequestImpl.send(HttpRequestImpl.java:77)
	at org.expath.httpclient.HttpClient.sendOnce(HttpClient.java:121)
	at org.expath.httpclient.HttpClient.doSendRequest(HttpClient.java:103)
	at org.expath.httpclient.HttpClient.sendRequest(HttpClient.java:89)
	at org.expath.httpclient.saxon.SendRequestCall.call(SendRequestCall.java:65)
	at net.sf.saxon.functions.IntegratedFunctionCall.iterate(IntegratedFunctionCall.java:297)
	at net.sf.saxon.expr.elab.FallbackElaborator.lambda$elaborateForPull$0(FallbackElaborator.java:46)
	at net.sf.saxon.expr.instruct.CopyOf$CopyOfElaborator.lambda$elaborateForPush$7(CopyOf.java:1057)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:376)
	at net.sf.saxon.trans.Mode.handleRuleNotNull(Mode.java:587)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:521)
	at net.sf.saxon.trans.XsltController.applyTemplates(XsltController.java:684)
	at net.sf.saxon.s9api.AbstractXsltTransformer.applyTemplatesToSource(AbstractXsltTransformer.java:430)
	at net.sf.saxon.s9api.Xslt30Transformer.applyTemplates(Xslt30Transformer.java:306)
	at net.sf.saxon.Transform.processFile(Transform.java:1389)
	at net.sf.saxon.Transform.doTransform(Transform.java:879)
	at net.sf.saxon.Transform.main(Transform.java:83)
```

**Comment by adamretter** [comment]
@ajbufort I just released version 1.5.1 which I think may have fixed this. Could you test it please?

**Comment by adamretter** [comment]
@ajbufort Great. Thanks for reporting.

Closed by https://github.com/expath/expath-http-client-java/commit/430b1aaa5b20cf984abed3fa1bc0487ca4228f21
